GBHFacebookImagePicker/Classes/Controller/GBHFacebookAlbumPicker.swift

```diff
@@ -54,6 +59,9 @@ class GBHFacebookAlbumPicker: UITableViewController {
 
         // Start Facebook login
         self.doFacebookLogin()
+
+        // Statusbar
+        self.setNeedsStatusBarAppearanceUpdate()
```

Comment:
I don't think this is useful. Maybe it's only required when you change a status bar propreties when the controller is already visible ?

Reply:
You are right. I just removed these lines